openreview/workflows/edit_invitations.py

```diff
@@ -1650,4 +1650,171 @@ def set_edit_submission_release_source_invitation(self, super_invitation_id, due
+                        }
+                    },
+                    'edit': {
+                        'note': {
+                            'content': {
+                                'reduced_load': {
+                                    'order': 6,
+                                    'description': 'Please select the number of submissions that you would be comfortable reviewing.',
+                                    'value': {
+                                        'param': {
+                                            #'type': 'integer',
+                                            'type': 'string',
+                                            'enum': ['${9/content/reduced_load_options/value}'],
+                                            'input': 'select',
+                                            'optional': True,
+                                            'deletable': True
+                                        }
+                                    }
+                                }                                
+                            }
+                        }
+                    },                
+                    'signatures': [venue_id]
+                }
+            }  
+        )
+
+
+        if process_file:
+            invitation.process = self.get_process_content(f'{process_file}')
+
+        if due_date:
+            invitation.duedate = due_date
+
+        self.save_invitation(invitation, replacement=False)
+
+        invitation_id = super_invitation_id + '/Response_Emails'
+
+        invitation = Invitation(
+            id = invitation_id,
+            invitees = [venue_id],
+            signatures = [venue_id],
+            readers = [venue_id],
+            writers = [venue_id],
+            edit = {
+                'signatures': [venue_id],
+                'readers': [venue_id],
+                'writers': [venue_id],
+                'content' :{
+                    'accepted_message_subject_template': {
+                        'order': 1,
+                        'description': 'Accepted response subject',
+                        'value': {
+                            'param': {
+                                'type': 'string',
+                                'input': 'text'
+                            }
+                        }
+                    },
+                    'accepted_message_body_template': {
+                        'order': 2,
+                        'description': 'Accepted response subject',
+                        'value': {
+                            'param': {
+                                'type': 'string',
+                                'input': 'textarea'
+                            }
+                        }
+                    },
+                    'declined_message_subject_template': {
+                        'order': 3,
+                        'description': 'Accepted response subject',
```

Comment:
Don't think this is critical right now but we should update these descriptions at some point.